CAIPs/caip-2.md

```diff
@@ -0,0 +1,111 @@
+---
+caip: 2
+title: Blockchain ID Specification
+author: Simon Warta (@webmaster128), ligi <ligi@ligi.de>, Pedro Gomes (@pedrouid)
+discussions-to: https://github.com/ChainAgnostic/CAIPs/pull/1, https://github.com/UCRegistry/registry/pull/13, https://ethereum-magicians.org/t/caip-2-blockchain-references/3612,
+status: Draft
+type: Standard
+created: 2019-12-05
+updated: 2019-12-12
+---
+
+## Simple Summary
+
+CAIP-2 defines a way to identify a blockchain (e.g. Ethereum Mainnet, Görli, Bitcoin, Cosmoshub)
+in a human readably, developer friendly and transaction-friendly way.
+
+## Abstract
+
+Often you need to reference a blockchain. For example when you want to state where some asset or smart contract is located. In Ethereum the [EIP155](https://eips.ethereum.org/EIPS/eip-155) chain ID is used most of the time. But with an Ethereum chain ID you cannot reference e.g. a Bitcoin or Cosmos chain.
+
+## Motivation
+
+The final trigger to create this CAIP (and the CAIP process itself) was a discussion around [EIP2256] at [Ethereum-Magicians](https://ethereum-magicians.org/t/eip-2256-add-wallet-getownedtokens-json-rpc-method/3600/14).
+Independently, the [Universal Chain Registry](https://github.com/UCRegistry) was created that needs properly specified chain identifiers at its core. A [discussion about the network ID format](https://github.com/UCRegistry/registry/pull/13) brought this group together with ChainAgnostic.
+
+## Specification
+
+The blockchain ID (short "chain ID") is a string designed to uniquely identify blockchains in a developer-friendly fashion.
+
+### Syntax
+
+The `chain_id` is a case-sensitive string in the form
+
+```
+chain_id:    namespace + ":" + reference
+namespace:   [-a-z]{3,16}
+reference:   [-a-zA-Z0-9]{3,47}
```

Comment:
It should change to 1 because for ethereum main this is: eip155:1

Reply:
Nice catch! In this case, we should also open up the cosmos battern to allow chain "x" to be encoded as "cosmos:x"

Reply:
Yes I agree! If you could add my name on the spec in author or reviewer that's cool also :)

Reply:
Done in 69f92ed